### PR TITLE
chore(common): bump limit for workflow-test-pr

### PR DESCRIPTION
**Pull request by Joshua-Anderson**
We now have 16 clusters available on weekdays, so the number needs a bump.

Bonus points for anyone who knows how to create a followup PR the only raises the cap when clusterator is active. 

Ping @jchauncey

### CODE REVIEW

File: common.groovy [hunk 1]
```diff
@@ -56,7 +56,7 @@ defaults = [
   ],
   maxBuildsPerNode: 1,
   maxTotalConcurrentBuilds: 3,
-  maxWorkflowTestConcurrentBuilds: 3,
+  maxWorkflowTestConcurrentBuilds: 13,
```
Comment (vdice) [inline common.groovy]: if this PR is meant for `workflow-test-pr`, I believe we would bump the value immediately below this one.  (This one I believe is for `workflow-test`)

**Comment by Joshua-Anderson** [comment]
@vdice Fixed!

**Comment by jchauncey** [comment]
That would be easy to do but requires us to run the seed job all the time

**Comment by jchauncey** [comment]
@Joshua-Anderson this should be fine. I was actually thinking about just removing the limit completely

**Comment by Joshua-Anderson** [comment]
I wouldn't entirely remove the limit because it is possible to queue up a lot of builds at once. Merging a SDK or CLI PR kicks off a rebuild of all the pull requests in the repository. If those are in high enough numbers, it can overwhelm the cluster capacity.

**Comment by jchauncey** [comment]
We could probably up the number of clusters we create to handle that case.

**Comment by helgi** [comment]
Why does the SDK / CLI need to rebuild all PRs?

**Comment by Joshua-Anderson** [comment]
A quirk (or is it a feature 😛) of the jenkinsfile build system is that instead of building the commit of the PR, it builds the commit of the pull request merged into master that github would generate if you merged the PR using the web UI. When somebody merges a PR, this merge commit changes and jenkins kicks off a new PR build to make sure the PR is passing with the new commit.

For reference Travis CI also uses the merge commit to build PRs, but it doesn't rebuild if a PR is merged.

**Pull request merged**